CONTRIBUTING.md

```diff
@@ -15,79 +15,78 @@ highly appreciated!
 
 ## Development environment
 
-The development environment allows contributors to test their changes to the
-extension. The local repository source is built and loaded into the Extension
-Development Host.
+Setting up a dev environment allows contributors to test changes to this
+extension. The source code is built, and the resulting package loaded into a
+special instance of VS Code.
 
-First, ensure that [Yarn](https://yarnpkg.com/) and
-[Visual Studio Code](https://code.visualstudio.com) are installed.
+First, ensure that [Visual Studio Code](https://code.visualstudio.com) and
+[Yarn](https://yarnpkg.com/) are installed.
 
-- Open this repository's root directory as a project in VS Code.
+- Open this repository as a project in VS Code.
 
-- Run `Tasks: Run Build Task` (Ctrl+Shift+b) to start the extension and webview
-  development servers (alternatively run `yarn dev-server` from the terminal).
+- Run `Tasks: Run Build Task` (Shift + Ctrl/Cmd + `b`) to start the extension
+  and Webview dev servers (alternatively, run `yarn dev-server` from the
+  _Integrated Terminal_).
 
-  > **Warning**: Having a separate (`.vsix`) version of the extension installed
+  > **Warning**: Having a separate `.vsix` version of this extension installed
   > may cause all kinds of chaos in your development environment.
```

Comment:
Hey @mattseddon 👋🏼 what is this warning meant to prevent? `Tasks: Run Build Task` doesn't seem to generate a .vsix file.

Reply:
Also, I think it's just `yarn dev` now?

Reply:
definitely dev-server: `"dev-server": "yarn turbo run dev --parallel",`

Reply:
If you have the extension installed (either through a `VSIX` or the marketplace) and start a debug session with the `dev-server` running you're going to be very confused as to what is going on.

Reply:
I have the extension installed via marketplace.

Shift + Cmd + `b` seems to use `yarn run dev`:

<img width="1072" alt="image" src="https://user-images.githubusercontent.com/1477535/176597587-8d355e80-b04c-47a9-a7fc-9a4679fd0bc6.png">

Ends with

```
webpack 5.73.0 compiled successfully in 5388 ms
No issues found.
```

If I instead run yarn `dev-server` from Terminal, the output looks a bit different but the ending is similar:

```
dvc-vscode-webview:dev: webpack 5.73.0 compiled successfully in 5754 ms
dvc-vscode-webview:dev: No issues found.
```

Start Debugging in either case seems to work fine. The dev host disables all installed extensions anyway:

<img width="1440" alt="image" src="https://user-images.githubusercontent.com/1477535/176597806-7201eb62-4de5-4756-8915-090f91a37a79.png">

Then again the same happens if I don't run any build task before F5 😅 I've no idea what's happening anyway.

Bottom line: are these instructions clear for extension developers?

Reply:
Then again the same happens if I don't run any build task before F5 😅 I've no idea what's happening anyway.

Bottom line: are these instructions clear for extension developers?

Reply:
> the same happens if I don't run any build task before F5

Ah, after deleting extension/dist F5 fails.

Reply:
> Start Debugging after building (with or without webview's webpack watch) seems to work fine. The dev host disables all installed extensions anyway:

This ties in with https://github.com/iterative/vscode-dvc/pull/1928#discussion_r910635641